starters/tailwind/tsconfig.json

```diff
@@ -17,7 +17,8 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react"
+    "jsx": "react",
+    "skipLibCheck": true
```

Comment:
i'm not in love this with line, and I'm not sure why it suddenly became an issue, I'll try to revert it in a TS upgrade PR